src/interface/report/Results/TimeFilter.tsx

```diff
@@ -1,18 +1,23 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 
 import TimeInput from './TimeInput';
 
-class TimeFilter extends React.PureComponent {
-  static propTypes = {
-    fight: PropTypes.object.isRequired,
-    isLoading: PropTypes.bool.isRequired,
-    applyFilter: PropTypes.func.isRequired,
-  };
+interface Props {
+  fight: any,
```

Comment:
You used a fight type earlier with more details, if you make that an interface and export it, you can use it here to have at least a minimal amount of type info.

Reply:
I wasn't sure where to put a fight definition like that so I just opted to use any here and update it at a later stage when somebody converts that part to TS. if you have an idea where you want the fight object to be exported from I can try to fit all the attributes it possesses into it

Reply:
Just put it in the first place you run into it and move it if/when you find a better location